docs/content/troubleshooting.md

```diff
@@ -59,56 +59,74 @@ The most common causes of this exception are `bool` and `System.Boolean`, both o
 
 **Cause**<br/>
 
-This is a benign exception that is thrown when building a worker while burst compilation is turned on. Your worker was successfully built despite this error. This occurs becayse Unity's burst compiler doesn't yet fully support cross compilation (Windows to Linux, for example).
+This is a benign exception that is thrown when building a worker while burst compilation is turned on. Your worker was successfully built despite this error. This occurs because Unity's burst compiler doesn't yet fully support cross compilation (Windows to Linux, for example).
 
 **Workaround**<br/>
 
-To remove the error message, disable Burst compilation by unchecking `Jobs > Enable Burst Compilation` in the Unity Editor.
+To remove the error message, disable burst compilation by unchecking **Jobs** > **Enable Burst Compilation** in your Unity Editor.
 
+<br/>
 ## Warnings
+<br/>
+#### <li> <b>I get several warnings when opening the project for the first time</b>
 
-#### Several warnings when opening the project for the first time
-
+<br/>
 **Cause**<br/>
-These warnings originate from code written by Unity, `com.unity.mathematics` or `com.unity.entities` for example. These warnings are benign so you can ignore them.
+These warnings originate from code written by Unity, for example; `com.unity.mathematics` or `com.unity.entities`. 
+
+**Fix**<br/>
+These warnings are benign so you can ignore them.
 
-#### Several warnings when deleting a GameObject or ECS entity that represents a SpatialOS entity
+<br/>
+#### <li> <b> I get several warnings when deleting a GameObject or ECS entity that represents a SpatialOS entity</b>
 
 **Cause**<br/>
-These warnings occur when a [GameObject]({{urlRoot}}/content/glossary#gameobject) or [ECS entity]({{urlRoot}}/content/glossary#unity-ecs-entity) that represents a SpatialOS entity is deleted before its corresponding SpatialOS entity leaves the worker's view. The GDK is designed so that the GameObjects and ECS entities that represent SpatialOS entities should only be deleted after the SpatialOS entity they represent has left that worker's view. ECS entities are deleted automatically by the GDK when their corresponding SpatialOS entities leaves that worker's view. In the case of GameObjects, the SpatialOS Runtime sends a message to the Unity worker when a SpatialOS entity leaves that worker's view, the GDK passes this message onto you. It's your responsibility to act upon these commands in your code.
+These warnings occur when a [GameObject]({{urlRoot}}/content/glossary#gameobject) or [ECS entity]({{urlRoot}}/content/glossary#unity-ecs-entity) that represents a [SpatialOS entity]({{urlRoot}}/content/glossary#spatialos-entity) is deleted before its corresponding SpatialOS entity leaves the [worker's view]({{urlRoot}}/content/glossary#worker-s-view). 
+
+The GDK expects a GameObject and ECS entity that represent a SpatialOS entity are only deleted after the SpatialOS entity they represent has left that worker's view: 
```

Comment:
```suggestion
The GDK expects that a GameObject and ECS entity that represent a SpatialOS entity are only deleted after the SpatialOS entity they represent has left that worker's view: 
```

- added "that" to set up the sentence structure (I had originally assumed it would go on to say something like "The GDK expects a GameObject to do XYZ")
- this talks about "a GameObject and ECS entity" and is in the plural, whereas line 88 says "a GameObject or ECS entity" and uses the singular. Is one of these wrong?